python/hail/tests/test_expr.py

```diff
@@ -1220,6 +1220,13 @@ def test_array_methods(self):
 
         self.assertEqual(hl.eval_expr(hl.flatmap(lambda x: hl.range(0, x), [1, 2, 3])), [0, 0, 1, 0, 1, 2])
 
+    def test_sorted(self):
+        self.assertEqual(hl.eval_expr(hl.sorted([0, 1, 4, 3, 2], lambda x: x % 2)), [0, 4, 2, 1, 3])
```

Comment:
is it deterministic that the result is in this order, even when some elements share an ordering key?

Reply:
Yes, sort should be stable.